Makefile.linux.mk

```
@@ -80,6 +80,7 @@ build/$1-%/lib/pkgconfig/capstone.pc: build/$1-env-%.rc build/.capstone-submodul
 			*-x86)       capstone_archs="x86"     ;; \
 			*-x86_64)    capstone_archs="x86"     ;; \
 			*-arm)       capstone_archs="arm"     ;; \
+			*-armbe8)     capstone_archs="arm"     ;; \
```

```suggestion
			*-armbe8)    capstone_archs="arm"     ;; \
```

(Should be kept aligned.)